README.md

```diff
@@ -4,6 +4,76 @@ The following repository, `forecasttools-py` is a _Python package for common pre
 
 NOTE: This repository is a WORK IN PROGRESS.
 
+---
+
+# Datasets Native To This Package
+
+NOTE: Information. Section of the data. How to instantiate. Reason for being. How it was created.
```

Comment:
Remove?

Reply:
Section or note? The NOTE regards contents of several descriptions I still need to afford.  I want something (either in README or vignette) reading what data is native to `forecasttools`. Open to suggestion here for how to go a little beyond text-documentation and perform some showcasing.

Reply:
I was referring to the note. You can keep it if you provide more context. Right now it is very vague.

Reply:
In my mind: _information_ is description of the dataset source (e.g., what is NHSN?); _section of the data_ is what subset (if not the full dataset) of the dataset is included in `forecasttools`; _how to instantiate_ is how to call the dataset with `forecasttools` and what it looks like (output display); _reason for being_ is the answer to why this dataset is included native in `forecasttools`; and _how it was created_ is an example of calling the methods in `data.py` to generate the dataset.